plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/refactoring/rename/JavaRefactoringRename.java

```diff
@@ -66,7 +67,7 @@
 import org.eclipse.che.ide.ui.dialogs.DialogFactory;
 
 /**
- * Class for rename refactoring java classes
+ * Class for recommend refactoring java classes
```

Comment:
Please, check changes in comments and docs, quite a lot changes like 'rename'->'recommend'

Reply:
These changes is my negligence. I have took your suggestions and modify most problems. But my ip-validation is failing, so I executed 'revert'. These is my fault and I am sorry about that. 
This review make me seeing more problems in my code. I will correct them as soon as possible.